Gordon360/Services/RecIM/MatchService.cs

```diff
@@ -177,7 +179,9 @@ public MatchExtendedViewModel GetMatchByID(int matchID)
                                     Email = _accountService.GetAccountByUsername(pt.ParticipantUsername).Email,
                                     Role = pt.RoleType.Description
                                 }),
-                            MatchHistory = _context.MatchTeam.Where(_mt => _mt.ID == mt.ID)
+                            MatchHistory = _context.MatchTeam.Where(_mt => _mt.TeamID == mt.TeamID && _mt.Match.StatusID == 6)
+                                .OrderByDescending(mt => mt.Match.StartTime)
+                                .Take(5)
```

Comment:
Presumably this means that you might not have the entire match history, right? Is there a reason that you are only taking 5 of them? It seems kind of arbitrary.